.github/workflows/demo-project.yml

```diff
@@ -0,0 +1,28 @@
+name: Check Demo Project
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: demo-project
+
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v3
+
+      - name: Configure Node and package manager
+        uses: actions/setup-node@v3.6.0
+        with:
+          node-version-file: "demo-project/package.json"
```

Comment:
What does it mean if the node requirement is to be >= than a value? Which version does it pick? We've used `.nvmrc` files everywhere else, maybe we should stick to those here as well? Let's not deviate too much between repos.